lib/logs.py

```diff
@@ -86,13 +86,14 @@ def setup(dir='logs', minLevel=logging.INFO):
     stream_handler.setLevel(minLevel)
     stream_formatter = logging.Formatter(CONS_FORMAT)
     stream_handler.setFormatter(stream_formatter)
-    logger.addHandler(stream_handler)
+    # Redirect console to log file
+    logger.addHandler(file_handler)
 
-    logging.getLogger("botocore").setLevel(logging.WARNING)
-    logging.getLogger("boto3").setLevel(logging.WARNING)
-    logging.getLogger("urllib3").setLevel(logging.WARNING)
-    logging.getLogger("googleapiclient.discovery").setLevel(logging.WARNING)
+    logging.getLogger("botocore").setLevel(logging.DEBUG)
```

Comment:
I think instead of setting levels on these modules to `logging.DEBUG` you can just remove their lines altogether.

The purpose of setting them to WARNING / ERROR was because our file handler was logging out all DEBUG level logs, but we found the DEBUG logs in these particular modules to be overly verbose, so we manually adjusted the level at which these modules output their logs to higher levels.

Since the file_handler logs out at DEBUG already, then we don't need to explicitly set these to debug now.